config/nunjucks.config.js

```diff
@@ -1,4 +1,6 @@
+import { SafeString } from 'nunjucks/src/runtime';
```

Comment:
By adding this, we need to move the `nunjucks` dependency as a non dev dependency, or it fails when pushing to CF.